lib/pages/full_post/comment_section.dart

```diff
@@ -44,11 +44,13 @@ class CommentSection {
       CommentListOptions(onSortChanged: store.updateSorting, sortValue: sort),
       // sorting menu goes here
       if (postComments != null && postComments.isEmpty)
-        const Padding(
-          padding: EdgeInsets.symmetric(vertical: 50),
-          child: Text(
-            'no comments yet',
-            style: TextStyle(fontStyle: FontStyle.italic),
+        _centeredWithConstraints(
```

Comment:
Small suggestion: You're making a bunch of unrelated changes in one PR.
It would be easier to review if you make changes like this (and the one to `ci.yaml`) in separate PRs in future.

Reply:
Yeah, will do. But its just a one liner ;)